### PR TITLE
Support `apns-collapse-id` via `collapse_id` kwarg

### PR DESCRIPTION
**Pull request by ezheidtmann**
Re #429 

This would appear to work but hasn't been tested. I can't figure out how to run existing tests. :upside_down_face:

### CODE REVIEW

**Comment by jamaalscarlett** [comment]
@ezheidtmann you run the tests using tox.  
```
pip install tox
```
then run using:
```
tox
```

**Comment by ekimia** [comment]
what's the status of this?

**Comment by ezheidtmann** [comment]
I'm running in prod and it appears to work. We are having trouble with local notifications *sometimes* not getting overwritten by our remote notification using the collapse id, but I believe that's a bug in our code not in the APNS code.

**Comment by ezheidtmann** [comment]
I rebased on latest master. Looks like existing tests pass; Travis failures appear unrelated.

File: push_notifications/apns.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def _apns_send(
 		except ValueError:
 			raise APNSUnsupportedPriority("Unsupported priority %d" % (priority))
 
+	notification_kwargs["collapse_id"] = kwargs.pop("collapse_id", None)
```
Comment (auvipy) [inline push_notifications/apns.py]: this part needs to be tested to avoid any regression.

**Comment by ezheidtmann** [comment]
Ok. Unfortunately I will not write this test any time soon.

**Comment by sirdodger** [comment]
We have the same issue and are running off a private fork to fix the issue.

**Comment by matthewh** [comment]
I've opened a new PR #523 and included a basic test to ensure the value is passed to apns2. If you were tracking this PR, can you please review #523 and tell me if that test is sufficient to accept the newer PR and close this?

**Pull request closed without merging**